webpack/webpack.common.js

```diff
@@ -5,80 +5,80 @@ const MiniCSSExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 
 const serverConfig = {
-  entry: [
-    './app/server.ts'
-  ],
-  target: 'node',
-  output: {
-    path: path.resolve(__dirname, '../build'),
-    filename: '[name].js'
-  },
-  resolve: {
-    extensions: ['.ts', '.js'],
-  },
-  externals: [ nodeExternals() ],
-  module: {
-    rules: [
-      {
-        test: /\.ts$/,
-        use: [
-          'ts-loader',
-        ]
-      }
-    ]
-  },
+        entry: [
+            './app/server.ts'
+        ],
+        target: 'node',
+        output: {
+            path: path.resolve(__dirname, '../build'),
+            filename: '[name].js'
+        },
+        resolve: {
+            extensions: ['.ts', '.js'],
+        },
+        externals: [nodeExternals()],
+        module: {
+            rules: [
+                {
+                    test: /\.ts$/,
+                    use: [
+                        'ts-loader',
+                    ]
+                }
+            ]
+        },
 };
```

Comment:
The diff here is about the size of indentation? Is this needed?

Reply:
Not needed but all these formatting changes are auto merged